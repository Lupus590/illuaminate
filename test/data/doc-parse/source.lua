--- @source /foo/bar:23

--- @source /foo/bar
