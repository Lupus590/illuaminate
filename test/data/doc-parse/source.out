((source source.lua[1:1-1:23])
 (custom-source (path foo/bar) (start_line 23) (end_line 23)))
((source source.lua[3:1-3:20])
 (custom-source (path /foo/bar) (start_line 1) (end_line 1)))
